layers/+completion/ivy/funcs.el

```diff
@@ -71,7 +71,7 @@
       (lambda (string &optional _pred &rest _unused)
         "Grep in the current directory for STRING."
         (if (< (length string) 3)
```

Comment:
I think it would be better to remove the condition of `if` too.

Reply:
I don't think so. There is an "else" part in the conditional unless we don't need that also.